content/MBC1.md

```diff
@@ -31,7 +31,7 @@ cartridge is removed from the Game Boy. Available RAM sizes are: 2KByte
 (at A000-A7FF), 8KByte (at A000-BFFF) and 32KByte (in form of four 8K
```

Comment:
I see some not-KiB here! There are a few more, let's fix that while we're in the area.

Reply:
Fixed.